build/yamls/flow-visibility/flow-visibility.yaml

```diff
@@ -0,0 +1,410 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app: flow-visibility
+  name: flow-visibility
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
Should the Grafana deployment be able to watch services across all namespace or only in its own namespace?

Reply:
only in its own namespace. As Grafana only need to access the Clickhouse services, which is created by `kind: "ClickHouseInstallation"`, under the same namespace.